services/svc/servicefabric/Cargo.toml

```diff
@@ -37,5 +37,3 @@ no-default-tag = []
 "7_1" = []
 "7_2" = []
 "8_0" = []
-"8_1" = []
-"8_2" = []
```

Comment:
File order doesn't work for a few services. A `sort` option would be useful, but I'm not sure it would be best for all. Will try.